src/bundle/Resources/public/js/scripts/core/adaptive.items.js

```diff
@@ -62,7 +73,8 @@
                 }
 
                 const lastItem = this.items[this.items.length - 1];
-                const isLastNonactiveItem = lastItem === activeItem ? i === this.items.length - 2 : i === this.items.length - 1;
+                const isLastNonactiveItem =
+                    lastItem === activeItem ? i === itemsWithoutForce.length - 2 : i === itemsWithoutForce.length - 1;
```

Comment:
personally I don't like conditional expression inside ternary operator, imo it's hard to read and understand

Reply:
Do you have a proposition on how to split it into a few lines?

Reply:
BTW This fragment was also discussed in comments here: https://github.com/ezsystems/ezplatform-admin-ui/pull/1780#discussion_r657869437